test/e2e/network/ingress.go

```diff
@@ -935,3 +939,176 @@ func detectNegAnnotation(f *framework.Framework, jig *e2eingress.TestJig, gceCon
 		framework.ExpectNoError(err)
 	}
 }
+
+var _ = SIGDescribe("Ingress API", func() {
+	f := framework.NewDefaultFramework("ingress")
+	/*
+		Release: v1.19
+		Testname: Ingress API
+		Description:
+		- The networking.k8s.io API group MUST exist in the /apis discovery document.
+		- The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
+		- The ingresses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
+		- The networking/ingresses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
+	*/
+
+	ginkgo.It("should support creating Ingress API operations", func() {
+		// Setup
+		ns := f.Namespace.Name
+		ingVersion := "v1beta1"
+		ingClient := f.ClientSet.NetworkingV1beta1().Ingresses(ns)
+
+		implementationPathType := networkingv1beta1.PathTypeImplementationSpecific
+
+		ingTemplate := &networkingv1beta1.Ingress{
+			ObjectMeta: metav1.ObjectMeta{GenerateName: "e2e-example-ing",
+				Labels: map[string]string{
+					"special-label": "myingress",
```

Comment:
specify a label specific to this test instance (probably using `f.UniqueName`) so our cluster-wide list/watch still filter to ingresses we created for this test